config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,3 @@
+Delayed::Worker.destroy_failed_jobs = false
+Delayed::Worker.max_attempts = 1
+Delayed::Worker.max_run_time = 8.hour
```

Comment:
das ist jetzt 8 statt 4, richtig? in der hoffnung, dass die 8 nicht erreicht werden, richtig?

Reply:
ja, in der Hoffnung das eigentlich 4 nun nicht mal mehr erreicht wird, da wir ja statt einem Job nun 4000 in der Warteschlange haben. jeder von denen sollte also kurz sein